### DISCUSSION TITLE
Quadratic Assignment from SciPy

### DISCUSSION THREAD

**Discussion by PaulWAyers** [theochem/procrustes#118] [Ideas]
We should probably look at quadratic assignment methods from SciPy as "back ends" for 2-sided permutation procrustes with one transformation. We have all that functionality (and more, I think) but we can at least use this as tests.

https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.quadratic_assignment.html#scipy.optimize.quadratic_assignment

What do you think @FarnazH and @fwmeng88 ?

**Comment by FanwangM** [comment]
Thanks for noticing this.

This can be useful for 2-sided permutation Procrustes with one transformation. What is more interesting is the [options ](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.show_options.html#scipy.optimize.show_options) argument which supports a lot of optimizers. So I think it's possible to add `method="scipy"` as the "back end" to add the support. The only thing I need to think more is how are we going to fit the  `options` argument in scipy into our package.

@PaulWAyers @FarnazH
